extensions/ql-vscode/src/remote-queries/remote-queries-manager.ts

```diff
@@ -42,6 +42,7 @@ export interface UpdatedQueryStatusEvent {
   status: QueryStatus;
   failureReason?: string;
   numRepositoriesQueried?: number;
```

Comment:
We use numXXX in some cases (`numRepositoriesQueried`) and XXXcount in others (`resultCount`). It'd be nice to use a convention. I prefer `count` so the comment is not really related to this PR I guess 😬 but we could change it later on.